spec/classes/clamav_spec.rb

```diff
@@ -5,6 +5,11 @@
     context "on #{os}" do
       let(:facts) do
         facts.merge(:environment => 'test')
```

Comment:
This should probably be facts.merge!(:environment => 'test') or move environment to the two below.

merge only returns the new hash without modifying the original, so the way it is now environment is lost...

merge! modifies the original hash